ProjectCars/CarCRUD.cs

```diff
@@ -40,24 +38,87 @@ public static List<Car> GetCars()
 
         public Car Create(Car car)
         {
-            
-            return null;
+            var cars = getCarsJsonFromFile();
+            var carsList = DeserealizeJsonFile(cars);
+            carsList.Add(car);
+            SerializeJsonFile(carsList);
+            //string carStr = car.ToString();
+            //File.AppendAllText(_path, carStr);
```

Comment:
Advice: never leave commented code. We can safely remove it. If it's old code we can get it back from Git history.